lib/models/database/database.dart

```diff
@@ -25,7 +24,7 @@ class Database extends _$Database {
         },
         onUpgrade: (Migrator m, int from, int to) async {
           print('schema changed from $from to $to ');
-          if (from >= 1 && from <= 9) {
+          if (from >= 1 && from <= 10) {

```

Comment:
If we want that number to be the previous version, then let's define a `prevSchemaVersion` so we reduce redundancy
_Please ensure the syntax I used is fine before committing, or edit in your IDE_
```suggestion
          int prevSchemaVersion = schemaVersion - 1;
          if (from >= 1 && from <= prevSchemaVersion) {
```

Reply:
We can also use the strict less-than, so it never reaches the current version
```suggestion
          if (from >= 1 && from < schemaVersion) {
```

Reply:
Will add this when I finalize the db version. There's multiple PRs right now that require it.